src/clients/sap_ha.rb

```diff
@@ -53,6 +54,10 @@ class SAPHAClass < Client
 
     def initialize
       log.warn "--- called #{self.class}.#{__callee__}: CLI arguments are #{WFM.Args} ---"
+      #Take care that corosync is enabled and running
+      corosync = Yast2::Systemd::Service.find('corosync')
```

Comment:
Same as https://github.com/yast/yast-sap-ha/pull/11/files#r942146359. But if we are pretty sure that the corosync service is there, it should not be a problem. Just take it into account if we decide at some point to change the dependency, or the service gets renamed or whatever.